app/tooling/sales-admin/commands/vault_init.go

```diff
@@ -0,0 +1,96 @@
+package commands
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/ardanlabs/service/foundation/vault"
+)
+
+const credentialsFileName = "/tmp/vault-init.json"
+
+// VaultInit sets up a newly provisioned vault instance
+func VaultInit(vaultConfig vault.Config) error {
+	var (
```

Comment:
Not a fan of var blocks, especially inside a function.  I rather have these separated

var initResponse vault.InitResponse

Let's not use pointer semantics for variable declaration and use var for zero value construction.

Reply:
Noticing this is not needed until line 41, so let's move it.

Reply:
Declare a variable as close to where it is needed.